src/d/actor/d_a_player_main.cpp

```diff
@@ -2086,7 +2087,8 @@ BOOL daPy_lk_c::checkScopeEnd() {
 
 /* 80112100-8011215C       .text setSubjectMode__9daPy_lk_cFv */
 void daPy_lk_c::setSubjectMode() {
-    /* Nonmatching */
+    dComIfGp_setPlayerStatus0(0, daPyStts0_UNK2000_e);
+    mDoAud_seStart(0x8FA);
```

Comment:
there's an enum in JAZelAudio_SE.h that can be used for the 6 seStart calls in this PR